supplements/quests-from-the-infinite-staircase/items.xml

```diff
@@ -0,0 +1,205 @@
+			<div class="reference">
+				<div element="ID_WOTC_DMG_ARTIFACT_TABLE" />
+			</div>
+		</description>
+		<setters>
+			<set name="keywords">random properties, minor beneficial, major beneficial, minor detrimental, action, reverse gravity, disintegrate, haste, flame strike, hold monster, fear, slow, prismatic wall, spells</set>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="rarity">Artifact</set>
+			<set name="attunement">true</set>
+		</setters>
+	</element>
+
+	<element name="Heretic" type="Magic Item" source="Quests from the Infinite Staircase" id="ID_WOTC_QIS_MAGIC_ITEM_HERETIC">
+		<description>
+			<p>The blasphemous weapon Heretic was created by a cult to steal power from good-aligned gods. The blade hungers to strike down servants of the Upper Planes and weaken the forces of good.</p>
+			<p class="indent">You gain a +3 bonus to attack and damage rolls made with this magic weapon. <em>Heretic</em> has 6 charges for the following properties; the sword regains 1d4 + 1 charges daily at dawn:</p>
+			<p class="indent"><strong><em>Destroy Devotion.</em></strong> Once per turn when you hit a creature with this weapon, you can expend 1 of its charges to attempt to render the target powerless. The target must succeed on a DC 17 Constitution saving throw or have the paralyzed condition until the start of your next turn. Celestials have disadvantage on the save.</p>
+			<p class="indent"><strong><em>Faith Hunter.</em></strong> While holding the sword, you can use an action to expend 1 or more of its charges to cast one of the following spells from it: <i>Detect Evil and Good</i> (1 charge), <i>Fly</i> (2 charges), or <i>True Seeing</i> (3 charges).</p>
+			<p class="indent"><strong><em>Sentience.</em></strong> <em>Heretic</em> is a sentient, chaotic evil weapon with an Intelligence of 17, a Wisdom of 17, and a Charisma of 15. It has hearing and darkvision out to a range of 120 feet.</p>
+			<p class="indent">The weapon can speak, read, and understand Common and Giant, and it can communicate with its wielder telepathically. Its voice is soft and deep but rises in a wild furor when it's aware Celestials or worshipers of good-aligned deities are present. While you are attuned to it, <em>Heretic</em> also understands every language you know.</p>
+			<p class="indent"><strong><em>Personality.</em></strong> <em>Heretic</em> craves the destruction of good-aligned gods and their supporters. The blade is condescending and snobby, especially in the company of priests and other pious folk, at whom it often directs sneering comments. When <em>Heretic</em> identifies its quarry, it erupts in a frenzied need for violence.</p>
```

Comment:
app doesn't seem to like when italicized or bold text ends and then starts again without any other text in-between, it seems to delete spaces for some reason.
![image included](https://github.com/user-attachments/assets/86132c6b-d19b-4a1e-8307-461139f49eb1)
```suggestion
			<p class="indent"><em><strong>Sentience.</strong> Heretic</em> is a sentient, chaotic evil weapon with an Intelligence of 17, a Wisdom of 17, and a Charisma of 15. It has hearing and darkvision out to a range of 120 feet.</p>
			<p class="indent">The weapon can speak, read, and understand Common and Giant, and it can communicate with its wielder telepathically. Its voice is soft and deep but rises in a wild furor when it's aware Celestials or worshipers of good-aligned deities are present. While you are attuned to it, <em>Heretic</em> also understands every language you know.</p>
			<p class="indent"><em><strong>Personality.</strong> Heretic</em> craves the destruction of good-aligned gods and their supporters. The blade is condescending and snobby, especially in the company of priests and other pious folk, at whom it often directs sneering comments. When <em>Heretic</em> identifies its quarry, it erupts in a frenzied need for violence.</p>
```